MinecraftClient/Settings.cs

```diff
@@ -700,6 +711,11 @@ public static void WriteDefaultSettings(string settingsfile)
                 + "# Inventory Handling NEED to be enabled first\r\n"
                 + "enabled=false\r\n"
                 + "threshold=6\r\n"
+                + "\r\n"
+                + "[AutoCraft]"
+                + "# Inventory Handling NEED to be enabled first\r\n"
+                + "# Enable terrainandmovements if you need to use crafting table\r\n"
+                + "enabled=false\r\n"
```

Comment:
Could be nice to allow setting `configpath` here, since everything could hold into a single file, the user might prefer to use a single `autocraft.ini` file so it's better to put it as a setting :)